network/templates/base/observation_view.html

```diff
@@ -35,7 +35,12 @@
         </thead>
         <tbody>
             <tr>
-              <td>{{ observation.satellite.norad_cat_id }} - {{ observation.satellite.name }}</td>
+              <td>
+                <a href="https://db.satnogs.org/satellite/{{ observation.satellite.norad_cat_id }}"
+                   target="_blank">
+                  {{ observation.satellite.norad_cat_id }}  - {{ observation.satellite.name }}
+                </a>
+              </td>
```

Comment:
Since this is an external link i would prefer we add an external link icon next to the text and use that as a link. We could use the `glyphicon glyphicon-new-window` from [glyphicons](http://getbootstrap.com/components/#glyphicons).